LambdaEngine/Source/Game/Multiplayer/Client/ClientUtilsImpl.cpp

```diff
@@ -0,0 +1,38 @@
+#include "Game/Multiplayer/Client/ClientUtilsImpl.h"
+
+#include "Networking/API/NetworkSegment.h"
+#include "Networking/API/NetworkStatistics.h"
+
+#include "Networking/API/IClient.h"
+
+namespace LambdaEngine
+{
+	ClientUtilsImpl::ClientUtilsImpl()
+	{
+
+	}
+
+	ClientUtilsImpl::~ClientUtilsImpl()
+	{
+
+	}
+
+	Entity ClientUtilsImpl::GetEntity(int32 networkUID) const
+	{
+		auto pair = m_NetworkUIDToEntityMapper.find(networkUID);
+		ASSERT(pair != m_NetworkUIDToEntityMapper.end());
+		return pair->second;
+	}
+
+	void ClientUtilsImpl::RegisterEntity(Entity entity, int32 networkUID)
+	{
+		auto pair = m_NetworkUIDToEntityMapper.find(networkUID);
+		ASSERT(pair == m_NetworkUIDToEntityMapper.end());
```

Comment:
```suggestion
		ASSERT(m_NetworkUIDToEntityMapper.find(networkUID) == m_NetworkUIDToEntityMapper.end());
```

Reply:
Or you can use `unordered_map::contains`:
```suggestion
		ASSERT(!m_NetworkUIDToEntityMapper.contains(networkUID));
```